src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -240,6 +242,15 @@ private void updateReport(final LocalDate dateToShow) {
             workButtonBox.getChildren().add(createDeleteWorkButton(w));
             final TreeItem<TableRow> workRow = new TreeItem<>(new WorkTableRow(w, workButtonBox));
             projectRow.getChildren().add(workRow);
+            if(w.getId()==model.activeWorkItem.get().getId()){
+               workButtonBox.getChildren().remove(0,3);
```

Comment:
can't you do a clear() or something like that? We want to remove all children (the number may change at some point in time).

Reply:
Changed it like you suggested